src/components/projects/wbs-details/project-container/work-package-summary/work-package-summary.test.tsx

```diff
@@ -36,6 +36,12 @@ describe('Rendering Work Package Summary Test', () => {
     expect(
       screen.getByText(`${endDatePipe(wp.startDate, wp.duration)}`, { exact: false })
     ).toBeInTheDocument();
+    wp.expectedActivities.slice(0, 3).map((expectedActivity) => {
```

Comment:
Since we don't care about the return values for these tests, you can just use forEach and the linter will also stop complaining

Reply:
Ah, forEach makes more sense in this case, will change